compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1792,6 +1789,7 @@ impl<'tcx> PlaceRef<'tcx> {
     /// a single deref of a local.
     //
     // FIXME: can we safely swap the semantics of `fn base_local` below in here instead?
+    // FIXME-FIXME: What is `base_local`?
```

Comment:
Cc @spastorino  who wrote this comment (supposedly, `base_local` still existed back then)

Reply:
Yea, remove those FIXMEs, I think we outrefactored them a while ago.